scripts/lib/libvirt/fixtures/cloud-admin-uefi.xml

```diff
@@ -56,4 +56,8 @@
     </memballoon>
 
   </devices>
+  <qemu:commandline>
+   <qemu:arg value='-global'/>
+   <qemu:arg value='driver=cfi.pflash01,property=secure,value=off'/>
```

Comment:
this is quite interesting, I think tianocore / ovmf is not using secure boot by default:

http://blog.hansenpartnership.com/uefi-secure-boot/